db/migrate/20230920213015_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :id
```

Comment:
Same here you should not define this columns, those columns are generated by the frameworks